SWIPs/swip-honey_to_money.md

```diff
@@ -9,43 +9,42 @@ Created: 31-07-2019
 ---
 
 ## Simple summary 
-Swarm needs a mechanism to allow fluctuation of the exchange rate between honey (Swarms internal accounting unit) and the currency used to pay. We propose an on-chain price oracle that returns an exchange rate + validUntil tuple.  
+Swarm needs a mechanism to allow fluctuation of the exchange rate between honey (Swarm's internal accounting unit) and the currency used to pay for transferring data. This SWIP proposes an on-chain price oracle that provides an exchange rate with an associated expiration.
 
 ## Abstract 
 The ability to update prices is required in order to:
 * Facilitate experimentation with the absolute price of honey (e.g. start at a low price and gradually increase);
 * Off-set fluctuations of the currency used to pay;
-* Lower the price of Swarm in the long-run as the costs of operating a Swarm node decreases.
+* Lower the price of Swarm in the long-run as the costs of operating a Swarm node decrease.
 
-We propose an on-chain price oracle that returns the current exchange rate when queried. The price-oracle will be initially managed by Swarm developers and stakeholders.
+The proposal is an on-chain price oracle that returns the current exchange rate when queried. The price oracle will be initially managed by Swarm developers and stakeholders.
 
-This SWIP is part of a series of SWIPs (but can be implemented on it's own). To see the full picture, please refer to [swip-message_to_honey](./swip-message_to_honey.md), [swip-multiple](./swip-honey_to_money.md), [swip-multiple_payment_processing_support](./swip-multiple_payment_processing_support.md) and the diagram below:
+This SWIP is part of a series of SWIPs (but can be implemented on its own). To see the full picture, please refer to [swip-message_to_honey](./swip-message_to_honey.md), [swip-multiple](./swip-honey_to_money.md), [swip-multiple_payment_processing_support](./swip-multiple_payment_processing_support.md) and the diagram below:
 ![SWIP_Diagrams.svg](./../assets/swip-honey_to_money/SWIP_Diagrams.svg)
```

Comment:
The diagrams are also quite hard to see, but I don't see of a way to split them now.

Reply:
What do you mean with hard to see? Do you mean this is an issue of the github UI, or the diagrams themselves?

Reply:
they look very small (on my browser, at least).

i am aware they can be expanded though